app/services/merge_duplicate_patients.rb

```diff
@@ -0,0 +1,194 @@
+class MergeDuplicatePatients
```

Comment:
How about `MergePatients`, because we can pass any arbitrary collection of patients in here? This class isn't responsible for identifying the dupes.

Reply:
We already have a different `MergePatientService` :/ How about `DedupPatients` or something that is not similar to the "merge" terminology?

Reply:
I was thinking along `CombinePatientsWithData` for the same reason.